pandas/core/frame.py

```diff
@@ -6652,10 +6652,12 @@ def corr(self, method='pearson', min_periods=1):
 
         Parameters
         ----------
-        method : {'pearson', 'kendall', 'spearman'}
+        method : {'pearson', 'kendall', 'spearman', callable}
             * pearson : standard correlation coefficient
             * kendall : Kendall Tau correlation coefficient
             * spearman : Spearman rank correlation
+            * callable: callable with input two numpy 1d-array
```

Comment:
"callable expecting two 1d ndarrays and returning a float" (does the callable get ndarrays or series?)

Reply:
ndarrays